monero-outreach-docs/translations/pt-br/website-sections/about_pt_br.md

```diff
@@ -0,0 +1,60 @@
+# Sobre Monero Outreach
+
+Um grupo independente
+
+Nós somos um time de criadores, escritores, técnicos, tradutores e visionários. Monero Outreach começou em Junho de 2018 e continua ganhando novos e talentosos contribuidores. Nós somos um grupo de trabalho da comunidade descentralizada do Monero e, desta forma, não reivindicamos nenhum nível de autoridade sobre o Monero - nós estamos aqui simplesmente para seguir o mandato que estipulamos para nós mesmos, porque nós amamos o Monero.
+
+## Mandato Monero Outreach
+
+1. Impulsionar uma maior adoção e conhecimento através de uma variedade de atividades relacionadas à divulgação - particularmente relações públicas, educação e marketing.
+2. Respeitar o trabalho feito pelos Desenvolvedores Core e pelos grupos de trabalho da comunidade, os princípios fundamentais do Monero e o espírito Monero, em todos os sentidos, para que nós não prejudiquemos a integridade do que do que torna o Monero excelente.
+3. Fazer do mundo um lugar melhor através do Monero. Nós somos motivados pela pergunta: "Como podemos preparar nossas crianças para viver nesse mundo, se nós não preparamos o mundo para nossas crianças?"
+
+
+## Nossa trajetória
+
+### FUNDAÇÃO
+
+- Criar sistemas de comunicação.
+- Estabelecer flux de trabalho.
+- Traçar objetivos de curto e longo prazo.
+- Desenvolver o website.
+- Entrevistar os usuários atuais e os potenciais para ganhar conhecimento relevante em estratégias futuras.
+- Atrair voluntários e contribuidores talentosos.
+- Familiarizar-se com o processo e as pessoas do projeto Monero.
+
+### ESTRUTURA
+
+- Desenvolver ferramentas para ajudar na divulgação; considerando usuários novos e atuais, comerciantes, investidores e jornalistas.
+- Fornecer exemplos de como nossos esforços agregam valor ao projeto e expandem a visão.
+- Arrecadar fundos, proteger patrocinadores e gerar receita internamente para apoiar profissionais adicionais e estabilidade de longo prazo.
```

Comment:
This phrase has a problem of verbal regency.

My sugestion: Arrecadar fundos, proteger patrocinadores, gerar receita internamente para apoiar profissionais adicionais e obter estabilidade a longo prazo.